app/javascript/components/widgets/EncounterCalendar.vue

```diff
@@ -1,13 +1,13 @@
 <template>
 	<div id="encounter-calendar">
-		<div v-if="partnerships.length > 1">
+		<div v-if="partnerships.length > 1 && selectedEncounters.length">
 			<v-select v-model="selectedPartners" multiple :options="availablePartners" label="display" :close-on-select="false" :no-drop="empty" :searchable="!empty">
 				<template v-slot:selected-option="opt">
 					<span><span :class="`partnership-${opt.index}`" class="partner-indicator"></span>{{opt.display}}</span>
 				</template>
 			</v-select>
 		</div>
```

Comment:
i found what im calling a bug and i think this is where it's occurring:

from the select tags, if i have one partner selected, and deselect this partner, i don't expect the select to disappear. from a ui perspective, i think it's just as reasonable to assume a user may deselect a partner before selecting a partner, but this doesn't allow for this:
https://gyazo.com/f65cc888f460bfa3547949bd6cf5d1cc

Reply:
yep, this is a bug I introduced. Thanks for catching it. I'll fix tomorrow